hw2/stackCalculator/Calculator.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using StackCalculator.Tests;
+
+namespace StackCalculator
+{
+    internal class Calculator
+    {
+        public static double Calculation(string inputString, string stackType)
+        {
+            IStack stack;
+            if (stackType == "0")
+            {
+                stack = new StackOnArray();
+            }
+            else
+            {
+                stack = new StackOnList();
+            }
```

Comment:
Это буквально противоречит условию, `Стековый калькулятор должен знать только про интерфейс стека (то есть вообще в коде класса «Стековый калькулятор» не должно быть ни одного упоминания конкретных реализаций стека, даже если очень хочется). `. И даже если бы так было можно, можно было проще:
```suggestion
            IStack stack = stackType == "0" ? new StackOnArray() : new StackOnList();
```
Вообще, переменные без начального значения должны напрягать.